self-run-agents/ami-build/azp-arm-large.json

```diff
@@ -0,0 +1,69 @@
+{
+  "variables": {
+    "azure_pipelines_token": "null"
+  },
+  "builders": [{
+    "type": "amazon-ebs",
+    "region": "us-east-1",
+    "source_ami_filter": {
+      "filters": {
+        "virtualization-type": "hvm",
+        "name": "ubuntu/images/*ubuntu-bionic-18.04-arm64-server-*",
+        "root-device-type": "ebs"
+      },
+      "owners": ["099720109477"],
+      "most_recent": true
+    },
+    "instance_type": "a1.4xlarge",
+    "ssh_username": "ubuntu",
+    "ami_name": "envoy-azp-arm-large-{{timestamp}}",
+    "run_volume_tags": {
+      "Project": "Packer"
+    },
+    "run_tags": {
+      "Project": "Packer"
+    },
+    "tags": {
+      "Project": "envoy-azp-arm-large"
+    },
+    "security_group_ids": [
+      "sg-0e26fd7adddb1b9fa"
+    ],
+    "encrypt_boot": true
+  }],
+  "provisioners": [
+    {
+      "type": "shell",
+      "inline": [
+        "sudo apt-get update",
+        "sudo apt -y install docker.io",
```

Comment:
The `docker.io` from ubuntu upstream doesn't have enough version, we need 19.03 for this repo (in the case). So docker have to be installed from https://docs.docker.com/engine/install/ubuntu/

Reply:
Okay I’ll change.